interface/main/calendar/add_edit_event.php

```diff
@@ -116,9 +126,16 @@
     $collectthis = $collectthis[array_keys($collectthis)[0]]["rules"];
 }
 ?>
+<?php $disabled = (!$edit && $GLOBALS['enable_group_therapy'] )?' disabled=true; ':'';?>
+<?php if($disabled) echo '<script>$( document ).ready(function(){
```

Comment:
Also, please be sure to test using the event editor with group therapy enabled but without gcalendar permission.